src/screens/RestaurantList/RestaurantList.test.tsx

```diff
@@ -0,0 +1,137 @@
+import { render, screen } from "@testing-library/react-native"
+import { createStaticNavigation } from "@react-navigation/native"
+import { createNativeStackNavigator } from "@react-navigation/native-stack"
+
+import * as hook from "../../services/restaurant/hook"
+
+import RestaurantList from "./RestaurantList"
+
+describe("RestaurantList component", () => {
+  // Mock the hooks and components used in RestaurantList
+
+  const mockRestaurantsResponse = {
+    data: [
+      {
+        name: "Bagel Restaurant",
+        slug: "bagel-restaurant",
+        images: {
+          thumbnail:
+            "node_modules/place-my-order-assets/images/3-thumbnail.jpg",
+          owner: "node_modules/place-my-order-assets/images/1-owner.jpg",
+          banner: "node_modules/place-my-order-assets/images/2-banner.jpg",
+        },
+        menu: {
+          lunch: [
+            { name: "Crab Pancakes with Sorrel Syrup", price: 35.99 },
+            { name: "Steamed Mussels", price: 21.99 },
+            { name: "Roasted Salmon", price: 23.99 },
+          ],
+          dinner: [
+            { name: "Truffle Noodles", price: 14.99 },
+            { name: "Spinach Fennel Watercress Ravioli", price: 35.99 },
+            { name: "Herring in Lavender Dill Reduction", price: 45.99 },
+          ],
+        },
+        address: {
+          street: "285 W Adams Ave",
+          city: "Detroit",
+          state: "MI",
+          zip: "60045",
+        },
+        resources: {
+          thumbnail: "api/resources/images/3-thumbnail.jpg",
+          owner: "api/resources/images/4-owner.jpg",
+          banner: "api/resources/images/1-banner.jpg",
+        },
+        _id: "5NVE3Z5MXxX3O57R",
+      },
+      {
+        name: "Brunch Barn",
+        slug: "brunch-barn",
+        images: {
+          thumbnail:
```

Comment:
Since we don't care if the image actually loads in the test, these could be short, dummy links like just "thumbnail.jpg", right? I don't think the test data needs to be realistic. Real URLs could change and we're probably not going to update tests when they do.

Don't change anything, go ahead and merge this. We can talk about how this data should look as we get closer to writing the teaching content.

Reply:
Yeah, I imagine there's a good chance we'll have to alter the tests for the sake of testing against their work towards the solution for each unit. As they are I don't think they will work in the long term because I'm not quite sure how what steps they'll take towards each solution I at least wanted to test the functionality being correct. And trimming down the way-too-large dummy data was something I was considering.

For now, I'll leave it as you suggest but if I have any ideas I'll try to supply them during future talks.